### ISSUE TITLE
Error: Health must be between 0 and 18.0, but was 20.0

### ISSUE THREAD

**Issue by fillnati** [DRE2N/DungeonsXL#813]
Hey. I downloaded your plugin (DungeonsXL 0.18-SNAPSHOT-968), I liked its functionality. But an error occurred while creating the map. The fact is that I also use plugins, like this: MMoCore, mythic mobs. They can enhance health. And just when you try to connect to a new card (cmd: /dxl create test; /dxl edit test), an error occurs in the server console.
I use the server version: paper-1.15.2

_P.S. I made translation this plugin on Russian. If you need it, i could send._

here is the error itself:

```
> [20:52:21 INFO]: fillnati issued server command: /dxl edit
> [20:52:26 INFO]: fillnati issued server command: /dxl edit test1
> [20:52:26 INFO]: -------- World Settings For [DXL_Edit_0] --------
> [20:52:26 INFO]: View Distance: 10
> [20:52:26 INFO]: Item Merge Radius: 2.5
> [20:52:26 INFO]: Item Despawn Rate: 6000
> [20:52:26 INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
> [20:52:26 INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
> [20:52:26 INFO]: Nerfing mobs spawned from spawners: false
> [20:52:26 INFO]: Zombie Aggressive Towards Villager: true
> [20:52:26 INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 End City: 10387313 Slime: 987234911
> [20:52:26 INFO]: Mob Spawn Range: 6
> [20:52:26 INFO]: Max TNT Explosions: 100
> [20:52:26 INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
> [20:52:26 INFO]: Cactus Growth Modifier: 100%
> [20:52:26 INFO]: Cane Growth Modifier: 100%
> [20:52:26 INFO]: Melon Growth Modifier: 100%
> [20:52:26 INFO]: Mushroom Growth Modifier: 100%
> [20:52:26 INFO]: Pumpkin Growth Modifier: 100%
> [20:52:26 INFO]: Sapling Growth Modifier: 100%
> [20:52:26 INFO]: Beetroot Growth Modifier: 100%
> [20:52:26 INFO]: Carrot Growth Modifier: 100%
> [20:52:26 INFO]: Potato Growth Modifier: 100%
> [20:52:26 INFO]: Wheat Growth Modifier: 100%
> [20:52:26 INFO]: NetherWart Growth Modifier: 100%
> [20:52:26 INFO]: Vine Growth Modifier: 100%
> [20:52:26 INFO]: Cocoa Growth Modifier: 100%
> [20:52:26 INFO]: Bamboo Growth Modifier: 100%
> [20:52:26 INFO]: SweetBerry Growth Modifier: 100%
> [20:52:26 INFO]: Kelp Growth Modifier: 100%
> [20:52:26 INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Other 64
> [20:52:26 INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1
> [20:52:26 INFO]: Experience Merge Radius: 3.0
> [20:52:26 INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true
> [20:52:26 INFO]: [WorldGuard] (DXL_Edit_0) TNT ignition is PERMITTED.
> [20:52:26 INFO]: [WorldGuard] (DXL_Edit_0) Lighters are PERMITTED.
> [20:52:26 INFO]: [WorldGuard] (DXL_Edit_0) Lava fire is PERMITTED.
> [20:52:26 INFO]: [WorldGuard] (DXL_Edit_0) Fire spread is UNRESTRICTED.
> [20:52:26 INFO]: [WorldGuard] Loaded configuration for world 'DXL_Edit_0'
> [20:52:26 ERROR]: null
> org.bukkit.command.CommandException: Unhandled exception executing command 'dxl' in plugin DungeonsXL v0.18-SNAPSHOT-968
> 	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[Paper.jar:git-Paper-318]
> 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[Paper.jar:git-Paper-318]
> 	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.dispatchCommand(CraftServer.java:761) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.PlayerConnection.handleCommand(PlayerConnection.java:1860) ~[?:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1668) ~[?:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.PacketPlayInChat.a(PacketPlayInChat.java:47) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.MinecraftServer.ba(MinecraftServer.java:1097) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:1090) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1051) ~[Paper.jar:git-Paper-318]
> 	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:966) ~[Paper.jar:git-Paper-318]
> 	at java.lang.Thread.run(Unknown Source) [?:1.8.0_251]
> Caused by: java.lang.IllegalArgumentException: Health must be between 0 and 18.0, but was 20.0. (attribute base value: 20.0, player: fillnati)
> 	at org.bukkit.craftbukkit.v1_15_R1.entity.CraftLivingEntity.setHealth(CraftLivingEntity.java:112) ~[Paper.jar:git-Paper-318]
> 	at de.erethon.dungeonsxl.player.DInstancePlayer.clearPlayerData(DInstancePlayer.java:77) ~[?:?]
> 	at de.erethon.dungeonsxl.player.DEditPlayer.<init>(DEditPlayer.java:55) ~[?:?]
> 	at de.erethon.dungeonsxl.command.EditCommand.onExecute(EditCommand.java:81) ~[?:?]
> 	at de.erethon.commons.command.DRECommandExecutor.onCommand(DRECommandExecutor.java:67) ~[?:?]
> 	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[Paper.jar:git-Paper-318]
> 	... 17 more
```

**Comment by Sataniel98** [comment]
Duplicate of #809.

If you have a translation, please feel free to PR it to the languages branch.

**Comment by fillnati** [comment]
check bug fix in #809?
